.travis_centos.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -eux
+
+# Install dependency packages and start mysqld service.
+sh .travis_setup_centos.sh
```

Comment:
I think this should go into the Dockerfile, so that you can benefit from layer caching of these dependencies.

Reply:
All right. Let me move the logic to the Dockefile.

Reply:
There is a problem to move the `.travis_setup_centos.sh` to `Dockerfile` RUN (that is executed on `docker build`'s timing).
We can not start mysqld daemon service in the process of `docker  build`. (Can not access the service).
So, I moved only the part to install dependency to the Dockerfile RUN part.
I add the modification as another commit to see the change easily for now.
How?
Thank you for the checking.